samples/bluetooth/peripheral_with_multiple_identities/README.rst

```diff
@@ -0,0 +1,190 @@
+   The identity associated with the connection is retrieved by calling :c:func:`bt_conn_get_info()` to obtain :c:member:`bt_conn_info.id`.
+   The work item (advertiser) associated with this identity is then submitted to the workqueue to restart the advertiser. 
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_with_multiple_identities`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Testing
+=======
+
+|test_sample|
+
+1. |connect_kit|
+#. |connect_terminal|
+#. Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+
+   There should now be multiple devices advertising with the name ``Nordic multi adv id`` followed by the ID.
+#. Connect to the device and observe the central's address is printed.
+#. Disconnect from any device and observe that the correct advertiser is disconnected.
+#. The disconnected advertiser should then be restarted.
+
+Alternatively, this sample can be tested with the sample :ref:`bt_scanning_while_connecting`.
+That sample will connect to all the advertising identities.
+
+Sample output
+=============
+
+The result should look similar to the following output::
+
+   I: SoftDevice Controller build revision:
+   I: d6 da c7 ae 08 db 72 6f |......ro
+   I: 2a a3 26 49 2a 4d a8 b3 |*.&I*M..
+   I: 98 0e 07 7f             |....
+   I: HW Platform: Nordic Semiconductor (0x0002)
+   I: HW Variant: nRF52x (0x0002)
+   I: Firmware: Standard Bluetooth controller (0x00) Version 58.49843 Build 3849690618
+   I: Identity: FA:BB:79:57:D6:45 (random)
+   I: HCI: version 6.0 (0x0e) revision 0x1055, manufacturer 0x0059
+   I: LMP: version 6.0 (0x0e) subver 0x1055
+   Bluetooth initialized
+   Starting 10 advertisers
+   Using current id: 0
+   Created Nordic multi adv id 0: 0x20007098
+   Advertiser 0 successfully started
+   New id: 1
+   Using current id: 1
+   Created Nordic multi adv id 1: 0x200070b0
+   Advertiser 1 successfully started
+   New id: 2
+   Using current id: 2
+   Created Nordic multi adv id 2: 0x200070c8
+   Advertiser 2 successfully started
+   New id: 3
+   Using current id: 3
+   Created Nordic multi adv id 3: 0x200070e0
+   Advertiser 3 successfully started
+   New id: 4
+   Using current id: 4
+   Created Nordic multi adv id 4: 0x200070f8
+   Advertiser 4 successfully started
+   New id: 5
+   Using current id: 5
+   Created Nordic multi adv id 5: 0x20007110
+   Advertiser 5 successfully started
+   New id: 6
+   Using current id: 6
+   Created Nordic multi adv id 6: 0x20007128
+   Advertiser 6 successfully started
+   New id: 7
+   Using current id: 7
+   Created Nordic multi adv id 7: 0x20007140
+   Advertiser 7 successfully started
+   New id: 8
+   Using current id: 8
+   Created Nordic multi adv id 8: 0x20007158
+   Advertiser 8 successfully started
+   New id: 9
+   Using current id: 9
+   Created Nordic multi adv id 9: 0x20007170
+   Advertiser 9 successfully started
+
+Extending the sample
+********************
+
```

Comment:
You should have some sort of introductory text here. Something like: "To get more out of this sample you can add some features as listed below:" And the items below are should not be numbered steps, as they are not steps.